opallabs/spredfast
# OP-142 Adds delete_message API call

## Context
This adds a delete_message API call using the new Spredfast delete API (https://spredfast-ext.jiveon.com/community/developer-center/conversations/messages). I named it delete_message rather than delete, because we had to add a new private delete method to mimic the pattern we were following with the other http verbs, and there was a naming conflict.

## Testing
- Build the new gem code locally
- Open up an irb session using the new gem code
- Create a client using the test credentials from dispatch.rake
- Post a message to spredfast and get the returned message id
- Try deleting the message! The status will be set to DELETED and it will no longer appear in the Spredfast UI
